### PR TITLE
🌱 Update OWNERS

### PR DESCRIPTION
**Pull request by Fedosin**
**What this PR does / why we need it**:

This PR creates a new group `cluster-api-operator-maintainers` to distinguish project maintainers from admins.

Also for numerous quality improvements we add `furkatgofurov7` as a project maintainer.

### CODE REVIEW

**Review by alexander-demicev — APPROVED** [review]
/approve

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -14,7 +14,14 @@ aliases:
   # active folks who can be contacted to perform admin-related
   # tasks on the repo, or otherwise approve any PRS.
   cluster-api-operator-admins:
+  - Fedosin
+  - alexander-demicev
+  - damdo
+
```
[the remark this answers is not in the document]
Comment (JoelSpeed) [reply OWNERS_ALIASES:20]: Yes, I still have admin access to the repository so can be left on the admins list

[the remark this answers is not in the document]
Comment (Fedosin) [reply OWNERS_ALIASES:20]: will be done

**Review by damdo — COMMENTED** [review]
Thanks
/lgtm

**Pull request merged**